config/sync/field.storage.paragraph.field_max_item_amount.yml

```diff
@@ -0,0 +1,69 @@
+uuid: a5e5cb11-e257-45dd-8fe1-b15f6f76847d
+langcode: en
+status: true
+dependencies:
+  module:
+    - options
+    - paragraphs
+id: paragraph.field_max_item_amount
+field_name: field_max_item_amount
+entity_type: paragraph
+type: list_string
+settings:
+  allowed_values:
+    -
+      value: '1'
+      label: '1'
+    -
+      value: '2'
+      label: '2'
+    -
+      value: '3'
+      label: '3'
+    -
+      value: '4'
+      label: '4'
+    -
+      value: '5'
+      label: '5'
```

Comment:
I didnt choose it - this is how the old `"field_amount_of_events"` does things.
The reason you dont see it in the diff, is that the old field still exists.

I'm not the one that has originally made the field, but, I believe the reason it was done this way, was to make it visually clear for the editor which options are available, so it can be shown in a dropdown.

The code looks weird, but the user experience works better than a `number input`

Reply:
:+1: Thanks for the explanation. 

Personally I am not keen on this solution but I guess it works.